siip.tex

```diff
@@ -78,48 +82,56 @@
           development efforts already tackled by colleagues. What's worse is 
           that these efforts are rarely, if ever, reviewed by, shared with, or 
           extended upon by peers.
+          %Q:  If the process of lesson preparation is valuable in ordering one's thoughts,
+          % doesn't an "on-tap" curriculum sap instructor effectiveness?
           
-          Open source software development provides an excellent 
-          example of a possible solution.
-          These communities have mastered distributed expert collaboration, 
-          dynamic peer review, and de-duplication of effort. In open source 
-          software, developers share code revisions in online repositories, 
+          Open-source software development suggests a possible collaborative solution.
+          These communities have mastered the challenges of distributed expert collaboration, 
+          dynamic peer review, and de-duplication of effort. In open-source 
+          software development workflows, developers share code revisions in online repositories, 
           review one another's work in small chunks \cite{wilson_best_2014}, 
           and contribute back their own improvements to the main project.
+          Thus open-source projects can focus concentration on particularly needed tasks
+          even from diffuse effort.
 
-          So, why aren't professors sharing their lesson materials online, 
+          So why aren't professors sharing their lesson materials online, 
           collaborating on canonical lesson sets, diffing and merging similar 
-          lessons, and reviewing one another's learning materials? \textbf{Our 
-          goal is to explore the possibility that curriculum development for 
-          university courses can operate like open source software development 
-          does.}
-           
+          lessons, and reviewing one another's learning materials?
           Previous attempts at collaborative curriculum development have met with 
-          challenges. Most notable among these is Software Carpentry (with 
-          which the PI and Co-PI are quite involved) 
-          \cite{wilson_software_2014}.  Software Carpentry teaches 
+          challenges, including instructor inertia, an unclear process for collaboration,
+          lack of a committed core team, differing visions, and particular institutional needs.
+          %Q:  Why?  If you raise the question, we need at least a stab at an answer here.
+          % I know for myself the fact that I work in triage is a major factor.
+          \textbf{Given our experience with successsful collaborative curriculum development
+          in niche areas, our goal is to explore the possibility that curriculum development for 
+          university courses can operate as well as open source software development 
+          does.}
+          
+          Most notable among successful efforts is Software Carpentry (a program
```

Comment:
Decidedly not. One of the key lessons learned from software carpentry is that it's failed to get as much collaboration as we expected. We had to force it at every step. See: lessons learned paper and Greg's scipy keynote, both cited here. Software carpentry faced challenges with this, that's what I'm saying here.